docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +162,203 @@ Note that:
+  - declaring _supported operations_ for configuration management: `c8y_UploadConfigFile` and `c8y_DownloadConfigFile`
+  - declaring provided _configuration types_
+
+**Declaring 'supported operations'**
+
+To declare supported operations the `c8y_configuration_plugin` uses thin-edge's _Supported Operations API_. Therefore the `c8y_configuration_plugin` creates for each child-device two files under `/etc/tedge/operations/c8y/<childid>`.
+
+Example, for child-device with childid `child1`:
+
+```
+/etc/tedge/operations/c8y/child1/c8y_UploadConfigFile
+/etc/tedge/operations/c8y/child1/c8y_DownloadConfigFile
+```
+
+As soon as those files are created, thin-edge's _Supported Operations API_ takes care to send according supported operation declarations to cloud's child-device twins (see [documentation Supported Operations](../tutorials/supported_operations.md#supported-operations-for-child-devices)).
+
+**Declaring 'provided configuration types'**
+
+For all configuration `types` provided by the external device, the plugin sends an MQTT message to C8Y. Thereby all `types` will be combined in one single message as below:
+  - topic: `c8y/s/us/<childid>`
+  - payload: `119,<type 1>,<type 2>,<type 3>,...`<br/>
+    Example: `119,foo.conf,bar.conf`
+
+Note that the `c8y_configuration_plugin` does **not** create any child-device twin in the cloud. Instead the clouds child-device twins must be created upfront.
+
+
+## Details to Aspect 2: Filetransfer from/to external device
+
+To provide/consume configuration files to/from external devices, the `c8y_configuration_plugin` make use of the HTTP filetransfer feature of the `tedge_agent` is used.
+  
+The HTTP filetransfer feature of the `tedge_agent` provides the service to transfer files from external devices to the local filesystem of the thin-edge device's, and vice versa.
+
+  * `tedge_agent` serves PUT and GET requests for temporary file up/downloads
+    * a temporary file can be uploaded with an HTTP PUT request to `http://<thin-edge IP address>/tedge/tmpfiles`,
+      where the HTTP response contains a _temporary URL_.
+    * a temporary file can be downloaded with an HTTP GET request to `http://<ip address of thin-edge device>/<temporary URL>`
+    * a local API allows the plugin to access (expose and obtains) a temporary file directly via the thin-edge device's local filesystem
+
+    * TODO: Investigate and decide about filetransfer's local API in scheduled prototype (https://github.com/thin-edge/thin-edge.io/issues/1307), according to topics below:
+ 
+      * local API must avoid that there are two copies of the same file at any point in time
+      * local API must avoid that the file content is transfered/copied from one to another place, when the plugin _obtains_ the file. 
+        Instead just the directory link may change (as it is the case for bash `mv` command).
```

Comment:
Not sure if this hard restriction should be a part of the specification as it assumes that the plugin and the file transfer service shares a file system. They could be running as separate containers as well.

Reply:
Removed now from specification, see https://github.com/thin-edge/thin-edge.io/pull/1236#discussion_r940987634